lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -898,6 +898,7 @@ def add_payment_method_or_subscription(xml, money, payment_method_or_reference,
           add_purchase_data(xml, money, true, options)
           add_installments(xml, options)
           add_creditcard(xml, payment_method_or_reference)
+          add_other_tax(xml, options)
```

Comment:
Adding the method call here makes it so that these fields are only passed on credit card transactions.

What about when the transaction is a check or a reference (third party token)? I don't see anything in the conversation about this change that indicates the tax fields should be excluded from those transactions.

Unless I am missing something, it seems to me like `add_other_tax` should be called from the `build_auth_request` and `build_purchase_request` methods.

Reply:
Right, this change is updated so that the `otherTax` is passed in for both request separately.